src/rtctools_heat_network/workflows/grow_workflow.py

```diff
@@ -450,7 +450,8 @@ def post(self):
         #             "fields": fields
         #         })
         # client.write_points(points=json_body, database=DB_NAME, batch_size=100)
-        self._write_updated_esdl(db_profiles=False)
+
+        self._write_updated_esdl()
```

Comment:
shall we delete the commented code above?

Reply:
I was wondering about this. I agree. It has been removed